### DISCUSSION TITLE
How to remove players with zero cash from the list?

### DISCUSSION THREAD

**Discussion by FrostBlade5** [EssentialsX/Essentials#4804] [Q&A] [answered]
### Type of bug

Other unexpected behaviour

### `/ess dump all` output

not about dumps

### Bug description

**Hey, i recently changed my server from public to whitelisted.
There is about 500 players who have a profil for money on the server.
I use killermoney and essentials plugins.

The command /balancetop shows a list of ALL players and how much money they have.
I made a script to delete the money status from all players to clear the list of inactive players.
However there are still 500 players on the list, their amount of money is ZERO.

How can i remove those players from the balancetop list?

Screenshot:**
https://i.postimg.cc/9m5ysY3L/2022-02-12-20-21-44.png

### Steps to reproduce

looking for help

### Expected behaviour

looking for help

### Actual behaviour

looking for help

**Comment by Chew** [comment, marked as answer]
If you're on the latest essentials, you can simply turn off [`show-zero-baltop`](https://github.com/EssentialsX/Essentials/blob/2.x/Essentials/src/main/resources/config.yml#L833)

**Reply by FrostBlade5** [reply]
> If you're on the latest essentials, you can simply turn off [`show-zero-baltop`](https://github.com/EssentialsX/Essentials/blob/2.x/Essentials/src/main/resources/config.yml#L833)

Where and how exactly can i do that?

**Reply by Chew** [reply]
The `config.yml`, if it's not there you just need to add it manually.

**Reply by FrostBlade5** [reply]
Thanks, will try that!
